.github/workflows/0-start-exercise.yml

```
@@ -0,0 +1,157 @@
+name: Step 0
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  actions: write
+  issues: write
+
+env:
+  STEP_1_FILE: ".github/steps/1-resolve-duplicate-issues.md"
+
+jobs:
+  start_exercise:
+    if: |
+      !github.event.repository.is_template
+    name: Start Exercise
+    uses: skills/exercise-toolkit/.github/workflows/start-exercise.yml@v0.3.0
+    with:
+      exercise-title: "Connect the Dots"
+      intro-message: "Let's learn to navigate a GitHub repository effectively!"
+
+  create_first_task: 
+    name: Create first task
+    runs-on: ubuntu-latest
+    needs: [start_exercise]
+    env:
+      ISSUE_URL: ${{ needs.start_exercise.outputs.issue-url }}
+    outputs:
+      original_issue: ${{ steps.issue_a.outputs.original_issue }}
+      duplicate_issue: ${{ steps.issue_b.outputs.duplicate_issue }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Create 'Fix broken sidebar' issue (target for duplicate)
+        id: issue_a
+        uses: actions/github-script@v7
+        with:
+          result-encoding: string
+          script: |
+            const { data: issue } = await github.rest.issues.create({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              title: "Fix the broken sidebar",
+              body: "The sidebar in `docs/_sidebar.md` is broken and needs fixing."
+            });
+            core.setOutput("original_issue", issue.number);
+
+      - name: Create 'Duplicate report' issue (to be closed as duplicate)
+        id: issue_b
+        uses: actions/github-script@v7
+        with:
+          result-encoding: string
+          script: |
+            const { data: issue } = await github.rest.issues.create({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              title: "Sidebar bug again",
+              body: "Looks like the sidebar bug was already reported."
+            });
+            core.setOutput("duplicate_issue", issue.number);
+
+
+  prep_repo_content: 
+    name: Prep repo for next step
+    runs-on: ubuntu-latest
+    needs: [start_exercise, create_first_task]
```

I don't see a dependency that this job needs to be run after `create_first_task`, or am  I wrong?
```suggestion
    needs: [start_exercise]
```